src/Administration/Resources/app/administration/src/module/sw-settings-services/component/sw-settings-services-service-card/sw-settings-services-service-card.spec.js

```diff
@@ -0,0 +1,279 @@
+import { mount } from '@vue/test-utils';
+import SwStatus from 'src/app/component/utils/sw-status';
+import {
+    MtModalAction,
+    MtModalRoot,
+    MtModal,
+    MtModalTrigger,
+    MtPopover,
+    MtPopoverItem,
+    MtButton,
+} from '@shopware-ag/meteor-component-library';
+import SwSettingsServicesServiceCard from './index';
+import SwColorBadge from '../../../../app/component/utils/sw-color-badge';
+
+describe('src/module/sw-settings-services/component/sw-settings-services-service-card.ts', () => {
+    beforeAll(() => {
+        Shopware.Service().register('shopwareExtensionService', () => ({
+            activateExtension: jest.fn(),
+            deactivateExtension: jest.fn(),
+        }));
+
+        Object.defineProperty(window, 'location', {
+            configurable: true,
+            value: { reload: jest.fn() },
+        });
+    });
```

Comment:
You should restore the original value after the tests (even though I don't know if it's necessary)

```suggestion
    const originalLocation = window.location;

    beforeAll(() => {
        Shopware.Service().register('shopwareExtensionService', () => ({
            activateExtension: jest.fn(),
            deactivateExtension: jest.fn(),
        }));

        Object.defineProperty(window, 'location', {
            configurable: true,
            value: { reload: jest.fn() },
        });
    });
    
    afterAll(() => {
        Object.defineProperty(window, 'location', { configurable: true, value: originalLocation });
    });
```